documentation/catalog/guides/deploy.md

```diff
@@ -25,25 +24,34 @@ the app is built and tagged, deploy production:
 
 1. Checkout the
    [infrastructure repository](https://github.com/wordpress/openverse-infrastructure)
-   and bump the catalog version with the `just bump prod catalog-airflow`
-   command.
-1. Once you've verified that no DAGs are running, update the value of
-   `running_dags_cleared` to `true` in the
-   [production module declaration](https://github.com/WordPress/openverse-infrastructure/blob/27c41ede9b24991909194e0a6477f6b11fceac0c/environments/prod/catalog-airflow.tf#L33).
-1. `just apply prod catalog-airflow` and verify the plan before deploying.
-1. Restore the value of `running_dags_cleared` back to `false`.
+   and bump the catalog version with the `just bump production airflow` command.
+1. `just ansible/playbook production airflow.yml -t airflow` and verify the plan
+   before deploying. Unless configuration variables are changing along with the
+   docker image version, the only change should be to the docker image tag in
+   the compose file. Run the playbook with `-e airflow_apply=true` to instruct
+   the playbook to actually apply any changes.
+
+- If _any_ DAGs are running, the playbook will not apply the changes and will
+  let you know that. If this happens, visit Airflow and confirm the list of
+  running DAGs. If they can be stopped, stop them. If they need to be waited
+  for, wait until they are done, then run the playbook again. If you must deploy
+  and cannot wait for the DAGs to finish (or, if they are deferred and cannot
+  finish), run the playbook with `-e airflow_force=true` to ignore the running
+  DAGs check.
+- See the [setup](#setup) section above for more information about when to
+  decide if it is okay to deploy when DAGs are running.
```

Comment:
Should we also have information here about how to apply changes to the actual EC2 instances (e.g. if launch template changes).